cereal/log.capnp

```diff
@@ -1252,6 +1252,28 @@ struct LiveLocationKalman {
   }
 }
 
+
+struct LivePose {
+  # More info on reference frames:
+  # https://github.com/commaai/openpilot/tree/master/common/transformations
+  orientationNED @0 : Measurement32;
+  velocityDevice @1 : Measurement32;
+  accelerationDevice @2: Measurement32;
+  angularVelocityDevice @3 : Measurement32;
+
+  inputsOK @4 :Bool = true;
```

Comment:
these defaults made sense for reading old logs, but think we want them to default to False now